dockerfiles/verify_packages/docker-compose.yml

```diff
@@ -58,5 +58,11 @@ services:
     depends_on:
       - request-replayer
 
+  tar_gz:
+    image: debian:buster
```

Comment:
To avoid running `install.sh` every time, would it make sense to run one of our existing Buster builds as root?

Reply:
Good point @SammyK! Originally the idea was that we do not want to use our development containers to do verification as we might 'overfix' things there. Instead we reproduce in all containers (you can check other scenarios in this very same folder) a path similar to what our users are doing to install php and the tracer.

I am totally open to changing this approach, but in case we need to discuss this as a separate topic from this PR and involving all the other scenarios we test, I guess?

Reply:
Good point. Thanks for clarifying!